lib/halfnarp.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
```

Comment:
If you want that, please open another issue. We can discuss it there.

For starters:
* There is no URL for halfnarp data.
* The halfnarp file I received by mail needs to be edited to be valid JSON.
* `halfnarp.rb` is not even a class and not meant to be reused.
* Dropping everything not above the 95 percentile, might result in unexpected data for smaller conferences.
* Only linking to related events within the same conference might not be what we want long term.